src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java

```diff
@@ -96,7 +101,8 @@ protected void setFrameSelect() {
 
     /**
      * Gets selected width resolution for video converter.
-     * @return int, width selected of Resolution.
+     *
+     * @return an int, width selected of Resolution.
```

Comment:
It would be better to put: " an int with selected resolution"

Reply:
ok ray, i correct it